cypress/e2e/auth/t0-signup.cy.ts

```diff
@@ -0,0 +1,97 @@
+describe('Sign up page test', () => {
+  it('should be able to visit app', () => {
+    cy.visit('sign-up')
+  })
+  /** TODO -The following test is skipped
+   * It will work once as this email is not used yet
+   * We need to re-seed between any test to make it work.
+   */
+  it('should create a new user and redirect to the home after signup', () => {
+    cy.visit('sign-up')
+
+    cy.get('input[name="organizationName"]').type('Company name')
```

Comment:
Just a suggestion here.

I'm wondering if we should always use `data-test` and avoid any other kind of selector.